docs/docs/integrations/platforms/huggingface.mdx

```diff
@@ -145,3 +145,25 @@ See a [usage example](/docs/integrations/tools/huggingface_tools).
 ```python
 from langchain.agents import load_huggingface_tool
 ```
+
+## More
+
+### Hugging Face prompt injection identification
+
+>It uses one of the specialized Hugging Face models to detect prompt injection attacks.
+> It is useful for when you need to ensure that prompt is free of injection attacks.
+> Default is the `laiyer/deberta-v3-base-prompt-injection` model.  
```

Comment:
Also this isn't true - the default is the deepset one. Would it be better to just have the prompt injection guide linked as a bullet point to not have to maintain this in two different places?

I'm not sure we want a full section for every feature that happens to use Hugging Face on this page. Would be better to just link it!

Reply:
Yes, usually, the provider page keeps just a short description and reference to the main example.
I've included it here because it is essential, that we can use not just one but many models for that.
Anyway, you have a good point. I'm removing the model name.